py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -1391,6 +1391,24 @@ def aten_ops_atanh(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.atan2.default)
```

Comment:
If you require one or both input tensors to be `TRTTensor`, consider using:
https://github.com/pytorch/TensorRT/blob/6b8ccbc69f05de3b6275cac4613735934fb559b0/py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py#L62-L66

Reply:
I add `enforce_tensor_types` for both input and other.